terraform/environments/oasys/README.md

```diff
@@ -24,11 +25,12 @@ https://oasys.service.justice.gov.uk/
 
 ### **Incident contact details:**
 
-The best contact point is through slack on (#ask-laa-ops)[https://moj.enterprise.slack.com/archives/CEL68S0LD]
+The best contact point is through slack on [#ask-digital-studio-ops](https://moj.enterprise.slack.com/archives/C6D94J81E) and [#shef_dba](https://moj.enterprise.slack.com/archives/C8NJZFNPN) or use both.
 
 ### **Service team contact:**
 
-As above. The concierge for the day will contact you directly.
+The concierge for the day will contact you directly if you contact them through slack on [#ask-laa-ops](https://moj.enterprise.slack.com/archives/CEL68S0LD).
```

Comment:
Should be [#ask-digital-studio-ops](https://moj.enterprise.slack.com/archives/C6D94J81E)
LAA don't support this